src/classroom-courses/android-app-development-using-kotlin-and-java.hbs

```diff
@@ -15,13 +15,86 @@
     {{> coursecontents}}
     {{> courseenroll}}
 	{{> courseschedule}}
-    {{> registration}}
+    {{!-- {{> registration}} --}}
+    <section>
+                                    <div class="text-center">
+                                        <h3>REGISTER NOW</h3>
+                                    </div>
+                                    {{!-- <div class="container widget-container" data-widget-id='112'>
+                                        <iframe class="payment-widget" frameborder="0"
+                                                src="https://students.codingblocks.com/payment/iframe?item_id=112"></iframe>
+                                    </div> --}}
+                                    <div class="container">
+                                        <div class="course-tabs">
+                                            <ul class="nav nav-tabs" role="tablist">
+                                                <li role="presentation" class="wide-tab active" id="tab-delhi"><a href="#delhi" aria-controls="home" role="tab" data-toggle="tab" aria-expanded="true">Delhi</a></li>
+                                                <li role="presentation" class="wide-tab" id="tab-noida"><a href="#noida" aria-controls="profile" role="tab" data-toggle="tab" aria-expanded="false">Noida (50% inaugural discount)</a></li>
+                                                <li role="presentation" class="wide-tab" id="tab-dehradun"><a href="#dehradun" aria-controls="profile" role="tab" data-toggle="tab" aria-expanded="false">Dehradun (50% inaugural discount)</a></li>
+                                                <li role="presentation" class="wide-tab" id="tab-dwarka"><a href="#dehradun" aria-controls="profile" role="tab" data-toggle="tab" aria-expanded="false">Dwarka</a></li>
+                                            </ul>
+                                            <div class="tab-content">
+                                                <div role="tabpanel" class="tab-pane active" id="delhi">
+                                                    {{!-- <div class="col-xs-12 text-center bg-secondary" style="height: 100%;">
+                                                        <p class="uppercase" style="margin-top: 100px;margin-bottom: 100px;">Seats full. Admissions closed.</p>
+                                                    </div> --}}
+                                                    <div class="widget-container" {{#if frame-id-pitampura-lp}}data-widget-id='{{frame-id-pitampura-lp}}'{{/if}}>
+														<iframe class="payment-widget" {{#if frame-id-pitampura-lp}}id="{{frame-id-pitampura-lp}}"{{/if}} frameborder="0" src="{{{registerLinkPitampuraLP}}}"></iframe>
```

Comment:
Instead of having multiple `iframe` of the same src, use only one iFrame and change the src